sql/parser/sql.y

```diff
@@ -54,8 +55,8 @@ package parser
   limit          *Limit
   targetList     TargetList
   targetListPtr  *TargetList
-  privilegeType  PrivilegeType
-  privilegeList  PrivilegeList
+  privilegeType  structured.Privilege
```

Comment:
Hrmm, I really liked that `sql/parser` was a leaf and didn't depend on other packages.